packages/flutter/lib/src/material/i18n/localizations.dart

```diff
@@ -204,7 +270,14 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
     "closeButtonTooltip": r"Close",
     "nextMonthTooltip": r"Next month",
     "previousMonthTooltip": r"Previous month",
+    "nextPageTooltip": r"Next page",
+    "previousPageTooltip": r"Previous page",
     "licensesPageTitle": r"Licenses",
+    "pageRowsInfoTitle": r"$firstRow–$lastRow of $rowCount",
+    "pageRowsInfoTitleApproximate": r"$firstRow–$lastRow of about $rowCount",
```

Comment:
numbers in english have to be localised, in particular wrt the use of thousands separators.

Reply:
I'm using NumberFormat from the intl package now.